tests/DFE.FindInformationAcademiesTrusts.CypressTests/cypress/e2e/regression/cookies.cy.ts

```diff
@@ -1,14 +1,90 @@
+import commonPage from "../../pages/commonPage";
 import cookiesPage from "../../pages/cookiesPage";
-describe("Testing the Cookie page and its options", () => {
+import navigation from "../../pages/navigation";
+
+describe('Cookie page and consent tests', () => {
 
     beforeEach(() => {
-        cy.login();
+        cy.login()
+        cy.visit('/cookies')
+    });
+
+    it('should check that both mandatory and optional cookies both exist after accepting optional cookies', () => {
+        cookiesPage
+            .acceptCookies()
+            .clickSaveChangesButton()
+
+        commonPage
+            .checkSuccessPopup('You’ve set your cookie preferences')
+
+        cy.visit('/trusts/details?uid=5712')
+
+        //check optional cookies exist
+        cy.getCookie('ai_user').should('exist')
+        cy.getCookie('ai_session').should('exist')
+        cy.getCookie('ai_authUser').should('exist')
+        cy.getCookie('_ga').should('exist')
+
+        //check mandatory cookies exist after saving
+        cy.getCookie('.FindInformationAcademiesTrust.CookieConsent').should('exist')
+        cy.getCookie('ASLBSA').should('exist')
+        cy.getCookie('ASLBSACORS').should('exist')
+        cy.getCookie('.AspNetCore.Antiforgery.VyLW6ORzMgk').should('exist')
```

Comment:
Just wondering is this name consistent? Seems like it could be some kind of generated string that might change

Reply:
As discussed with Nick this appears to be stable and the same across our FIAT sites - I'm going to raise a tech debt ticket to update the name to something more static.